skt/reporter.py

```diff
@@ -723,7 +738,10 @@ def report(self):
 
         # Add the most basic parts of the email message
         if self.subject:
-            msg['Subject'] = self.subject
+            # Strip all leading Re: s
+            subject = self.subject.replace('re: ', '', re.IGNORECASE)
```

Comment:
Testing it with subject `Re: [PATCH] CONFIGURE: add xxx`, I'm getting `Re: [<get_status output here>] Re: [PATCH] CONFIGURE: add xxx` as new subject, which we probably don't want. This is because the normal `replace` method is not case-insensitive, and what you're specifying as the last parameter is actually the [max number of replacements](https://www.tutorialspoint.com/python/string_replace.htm). What you want for case-insensitive replacements is [`re.sub`](https://docs.python.org/3/library/re.html#re.sub). BUT, doing so with the testing subject will remove the `RE:` in `CONFIGURE:` as well, so you'd need to write a more complicated regex to only remove `re:` from before the square brackets.

Reply:
This won't catch all the possible language-specific "Re:" incarnations, such as "AW:" and "На:", but OK.

Here's the regex you need, I think:

    subject = re.sub("^ *(Re: *)*",
                     "Re: [" + self.get_status() + "] ",
                     self.subject,
                     flags=re.IGNORECASE)

Reply:
We are the ones passing the subject with `Re: ` in the pipeline, so I think that part should be OK